src/styles/icons.less

```diff
@@ -51,7 +51,7 @@
 .icon-context--first:before {
 	content: "\e90a";
 }
-.icon-context-last:before {
+.icon-context--last:before {
```

Comment:
you can't do this.. not until Idan/Carmel fixes the fonts.. (this is a generated file)

For now either create a new class in the context menu file that extends this one, or add an `icon` property to the context items..